src/klipse/utils.cljs

```diff
@@ -126,10 +126,10 @@
   (set! secured-eval true)
   (let [original-eval js/eval]
     (j/assoc! js/window :eval (fn [src]
-                        (original-eval (str "with (klipse_eval_sandbox){ " src "}"))))
+                        (original-eval (str ";(function(){with (this){ " src "}}).call(klipse_eval_sandbox)"))))
     (set! eval-in-global-scope js/eval)
     (j/assoc! js/window :klipse_unsecured_eval original-eval)
-    (j/assoc! js/window :klipse_eval_sandbox (clj->js (zipmap the-forbidden-symbols (repeat {}))))
+    (j/assoc! js/window :klipse_eval_sandbox (clj->js (zipmap the-forbidden-symbols (repeat (js/Object.create nil)))))
```

Comment:
@viebel I think this one makes sense on its own as a change; I'm not entirely sure of the security implications though it does unhinge the forbidden symbols from the `Object` and `Function` prototypes…